openfe/protocols/openmm_md/plain_md_methods.py

```diff
@@ -431,26 +431,29 @@ def _run_MD(simulation: openmm.app.Simulation,
             mc_steps=1,
         )
 
-        simulation.reporters.append(XTCReporter(
-            file=str(shared_basepath / output_settings.production_trajectory_filename),
-            reportInterval=write_interval,
-            atomSubset=selection_indices))
-        simulation.reporters.append(openmm.app.CheckpointReporter(
-            file=str(shared_basepath / output_settings.checkpoint_storage_filename),
-            reportInterval=checkpoint_interval))
-        simulation.reporters.append(openmm.app.StateDataReporter(
-            str(shared_basepath / output_settings.log_output),
-            checkpoint_interval,
-            step=True,
-            time=True,
-            potentialEnergy=True,
-            kineticEnergy=True,
-            totalEnergy=True,
-            temperature=True,
-            volume=True,
-            density=True,
-            speed=True,
-        ))
+        if output_settings.production_trajectory_filename:
```

Comment:
@hannahbaumann - I was thinking, do you want to just make this into a separate PR? I feel like we could merge this right now with little to no review.

Reply:
Will do!